template/lambda-sqs-worker-cdk/infra/appStack.ts

```diff
@@ -74,6 +72,7 @@ export class AppStack extends Stack {
       target: 'node20',
       // aws-sdk-v3 is set as an external module by default, but we want it to be bundled with the function
       externalModules: [],
+      nodeModules: ['@seek/logger', 'skuba-dive'],
```

Comment:
Added these because I got the following error and I am not sure what is causing it when i run `pnpm run deploy`. Happy to fix this if anyone happens to know why

```
[ERROR] Could not resolve "@seek/logger"

    src/app.ts:1:25:
      1 │ import createLogger from '@seek/logger';
        ╵                          ~~~~~~~~~~~~~~

  The Yarn Plug'n'Play manifest forbids importing "@seek/logger" here because it's not listed as a
  dependency of this package:
```

Reply:
```suggestion
```

Based on the error, I suspect you created a project with yarn >v1? I haven't seen this error but have used the template and haven't found this problem. Could you try with yarn 1 or pnpm?

Reply:
Might be related? https://github.com/seek-oss/skuba/issues/996#issuecomment-2319598106

Reply:
Thanks for that. Tried with `pnpm` and `yarn 1` and all looks good